.github/workflows/build.yaml

```diff
@@ -119,6 +119,23 @@ jobs:
         openmw-osg
         sdl2
 
+    - name: Move pdb files
+      working-directory: 'C:/vcpkg'
+      shell: bash
+      run: |
+        find installed -type f -name '*.pdb' | xargs dirname | sort -u | xargs -I '{}' -i mkdir -p 'pdb/{}'
+        find installed -type f -name '*.pdb' | xargs -I '{}' -i mv '{}' 'pdb/{}'
```

Comment:
This doesn't feel like a clear way to do this to me. I had to spend longer looking at it to be sure what it was doing than I'd like. Perhaps
```bash
rsync -a --prune-empty-dirs --include='*/' --include='*.pdb' --exclude='*' /installed /pdb
```
would be clearer, but it's still a Bashism on Windows, which is weird. I tried coming up with a simple PowerShell command, and ran into an unreasonable number of problems (I could have done it, but it wouldn't have been miles off the Bash version that initiated this complaint), but the classic workaround is a basic `robocopy`:
```powershell
robocopy installed pdb *.pdb /S /MOVE
```

Reply:
Before PowerShell gave access to a shell that could actually do anything without being horribly abused, CPM/DOS/Windows went through a lot of commands to copy files around so you could do all the things you might need without having to get the shell involved. It's generally helpful to remember `xcopy` and `robocopy` are there on Windows if something that seems like people would have done before you isn't simple.

Reply:
If everything could be so simple. `robocopy` returns `1` by default which makes the job fail because non zero exit code is considered to be an error: https://github.com/elsid/openmw-deps-build/actions/runs/9224772965/job/25380756487 .